### PR TITLE
Fix for issue #5816

### PR DESCRIPTION
**Pull request by gopkumr**
### Prerequisites
PR for the fix for Issue #5816 

**Steps to verify the fix**
Navigate to Umbraco > Settings >Document Type > Choose a document type
Select Templates from the top right corner

Verify the text under the template name and  extra template

All karma unit tests passed 

### Description
To add a space between the translated text and the text that is injected via binding a &nbsp; was added in between the text in the markup.
Please find below the screenshot after fix

Fix 5816

### CODE REVIEW

File: src/Umbraco.Web.UI.Client/src/views/components/umb-grid-selector.html [hunk 1]
```diff
@@ -24,15 +24,15 @@
...
+                <div class="umb-grid-selector__item-label -blue" ng-if="defaultItem !== null"><localize key="grid_chooseExtra">Choose extra</localize>&nbsp;{{ itemLabel }}</div>
+                <div class="umb-grid-selector__item-label -blue" ng-if="defaultItem === null"><localize key="grid_chooseDefault">Choose default</localize>&nbsp;{{ itemLabel }}</div>
             </div>
         </a>
 
     </div>
 
     <div class="text-center" ng-if="(availableItems | compareArrays:selectedItems:'alias').length === 0">
-        <small><localize key="general_all">Akk</localize> {{itemLabel}}s <localize key="grid_areAdded">are added</localize></small>
+        <small><localize key="general_all">Akk</localize>&nbsp;{{itemLabel}}s <localize key="grid_areAdded">are added</localize></small>
```
Comment (ronaldbarendse) [inline src/Umbraco.Web.UI.Client/src/views/components/umb-grid-selector.html]: The default text for `general_all` also isn't English:
```suggestion
        <small><localize key="general_all">All</localize>&nbsp;{{itemLabel}}s <localize key="grid_areAdded">are added</localize></small>
```

Comment (gopkumr) [reply src/Umbraco.Web.UI.Client/src/views/components/umb-grid-selector.html]: I did not realise the Akk is a typo. I will update that as well

**Comment by gopkumr** [comment]
@ronaldbarendse Updated the PR with option to retain spaces between text and tag during minification. The PR is out for review.

**Comment by poornimanayar** [comment]
Hi @gopkumr ,

Congratulations on your first PR. Once this PR is merged you will have a new "C-Trib" badge on your Our.Umbraco profile. So make sure you create one if you dont have one yet. We hope to see you around more often here now on!

I have given this PR a quick spin and the issue in question seems to have been addressed, I can see that the spaces have been inserted. Is there anything else I need to be testing? Can you think of anything else and let us know here?

Regards,
Poornima, [Pull Request team](https://umbraco.com/blog/introducing-the-new-cpr-team/)

**Comment by nul800sebastiaan** [comment]
Hey there @gopkumr - apologies for the delay! This is looking great now, thanks very much for the help!

And congrats on your first accepted PR for Umbraco CMS, great work! 👍 

You are now the proud owner of this shiny contributor badge on Our Umbraco! https://our.umbraco.com/members/id:276528/ 🎖

**Comment by gopkumr** [comment]
@poornimanayar @nul800sebastiaan  Sorry for delay in response. Thank you so much for verifying and merging my PR and giving me a contributor badge. Proud moment,

**Pull request merged**